internal/server/device/device_utils_network.go

```diff
@@ -337,6 +337,11 @@ func networkCreateTap(hostName string, m deviceConfig.Device) (uint32, error) {
 		}
 
 		mtu = uint32(nicMTU)
+
+		err = NetworkSetDevMTU(hostName, mtu)
+		if err != nil {
+			return 0, fmt.Errorf("Failed to set the MTU %d: %w", mtu, err)
+		}
```

Comment:
So the comment above remains true, when dealing with a bridged interface we need to make sure not to set a low MTU on the device as the bridge will change its own MTU to the lowest MTU present.

Setting values higher than the bridge's own value is usually not an issue though.

Reply:
if I understand correctly, I need to check the parentMTU and only set if it’s higher or none? in my case with OVN/OVS, parentMTU is empty.

Reply:
Yeah, that's right if `m["parent"]` is set, then we should only set the device MTU if it's higher than the parent. If `m["parent"]` isn't set, then we can just set the provided MTU.